lib/conjur/command/users.rb

```diff
@@ -136,6 +136,19 @@ class Conjur::Command::Users < Conjur::Command
       end
     end
 
+    user.desc 'Rotate the API key of the logged-in user'
+    user.command :rotate_api_key do |c|
+      c.action do |_global, _options, _args|
+        username, password = Conjur::Authn.read_credentials
+        new_api_key = Conjur::API.rotate_api_key(username, password)
```

Comment:
It's also possible to rotate the API key of another user (or host), if the current authenticated user has the necessary privilege. See APIary.